### ISSUE TITLE
-Wunused-but-set-variable in fs/btrfs/free-space-cache.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1672]
```
fs/btrfs/free-space-cache.c:929:24: warning: variable 'mapping' set but not used [-Wunused-but-set-variable]
        struct address_space *mapping;
                            ^
1 warning generated.
```

Patch submitted: https://lore.kernel.org/20220722163854.1189931-1-nathan@kernel.org/

**Comment by nickdesaulniers** [comment]
sounds like this got picked up?

**Comment by nathanchance** [comment]
Yes, my diff was squashed into the original change: https://git.kernel.org/kdave/c/87d1edac403607bab93021afc48b05c234b3028a

**Comment by nathanchance** [comment]
Now fixed in next-20220726.
